mkosi/manifest.py

```diff
@@ -64,6 +64,20 @@ def report(self) -> str:
             t += f"""\nChangelog:\n{self.changelog}\n"""
         return t
 
+def parse_pkg_desc(f: Path) -> Tuple[str, str, str, str]:
+    with f.open() as desc:
+        for line in desc:
+            line = line.strip()
+            if line == "%NAME%":
+                name = next(desc).strip()
+            elif line == "%VERSION%":
+                version = next(desc).strip()
+            elif line == "%BASE%":
+                base = next(desc).strip()
+            elif line == "%ARCH%":
+                arch = next(desc).strip()
+                break
+    return name, version, base, arch
```

Comment:
```suggestion
    return name, version, base, arch

```
while looking at what pyright is complaining about, I noticed missing newlines in between the functions. We follow a two newlines between functions style, like PEP8.